packages/admin/cms-admin/src/contentScope/ContentScopeIndicator.tsx

```diff
@@ -1,71 +1,76 @@
-import { css } from "@mui/material";
+import { messages } from "@comet/admin";
+import { Domain } from "@comet/admin-icons";
+import { SvgIconProps, useTheme } from "@mui/material";
 import { styled } from "@mui/material/styles";
 import React from "react";
+import { FormattedMessage } from "react-intl";
+
+const bgGrey = "#5A697E";
```

Comment:
This should come from the theme IMO

Reply:
Where should I add it to the theme? The color codes in the neutrals palette seem to be chosen systematically (@jamesricky?). 

Also, currently this color code is only used here. I don't want to "trash" the theme with a color code that's only used once.

Reply:
It seemed to me that this color is used more commonly in Admin applications. We could ask UX whether they plan to used it more commonly, and how they'd call it.

Reply:
I just asked UX. They said currently this color is only used here. But now we are debating changing the color to one of the greys in the theme

Reply:
The color is added to the theme in https://github.com/vivid-planet/comet/pull/1840